bin/ubuntu-core-initramfs

```diff
@@ -21,13 +115,32 @@ def path_join_make_rel_paths(path, *paths):
 
 
 # Include kernel modules specified in conf_file
-def add_modules_from_file(dest_d, kernel_root, modules_d, fw_d, conf_file):
+def add_modules_from_file(dest_d, kernel_root, modules_d, fw_d, conf_file, db,
+                          allow_discoverable=True):
```

Comment:
Some description of the function parameters (old&new) would be nice to have here.

Reply:
Done